apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -1745,4 +1746,51 @@ export const notImplementedCards: DbCard[] = [
 			afterDamage: removeIfUsedInCombat,
 		},
 	},
+	{
+		id: '114',
+		name: 'Greedy Fire',
+		type: 'spell',
+		colors: ['red'],
+		attack: {
+			label: '2X',
+			getValue: ({ game, ownerSide }) => {
+				const owner = game.board.players[ownerSide];
+				const cardsInHand = owner.hand.length;
+				return 2 * cardsInHand;
+			}
+		},
+		description: 'On reveal, return the top card of the stack this card will be placed. This card\'s attack is equal to the number of cards in your hand X 2',
+		effects: {},
+	},
+	{
+		id: '115',
+		name: 'Quenching Water',
+		type: 'spell',
+		colors: ['blue'],
+		attack: {
+			label: '5X',
+		},
+		description: 'On reveal, discard cards from your hand. This card\'s attack is equal to the number of cards discarded X 5',
```

Comment:
that's sick, but very hard to implement, as now we'd have to keep a "custom state" to save the number of cards discarded, or have a form of "counters" like magic to keep the atk boost on a per-card basis! needs thinking architecturally

Reply:
Sounds good. We'll keep it in the "un-implemented" cards until we can spend time on this. I like the mechanic and am trying to push boundaries a little so we can mature our engine. This will be a stretch goal at some point in the future